x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/1000_rrf_retriever_highlighting.yml

```diff
@@ -0,0 +1,107 @@
+setup:
+  - requires:
+      cluster_features: 'rrf_retriever_composition_supported'
+      reason: 'test requires rrf retriever composition support'
+
+  - do:
+      indices.create:
+        index: test
+        body:
+          settings:
+            number_of_shards: 1
+            number_of_replicas: 0
```

Comment:
Why do you set replicas to `0`?

Reply:
Tbh the skeleton for the tests was copy-pasted from other tests, but I think that there were 2 reasons: first is to ensure consistency on the expected results since we rely on docids, and another one is that, IIRC, there was an issue with serverless and had to update this in some cases (looking for discussion/issue but haven't found it yet :/ )

Reply:
So, setting replicas: 0 (last I remember), would make serverless have no search nodes. We need search nodes as we do searches. Same goes for the other tests.

Reply:
Maybe I remember this for a different solution where we needed to disable some tests? 🤔 Not sure; and it's been a looong day :)  Will check tomorrow just for the sake of completeness, but happy to remove the setting.

Reply:
(provided ofc that the doc ids will not cause issues, given the small number of docs we actually index)